### PR TITLE
LoadToken

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

**Comment by erikzhang** [comment]
Go?

**Comment by cloud8little** [comment]
Test passed with following scenario:
1. deploy simple contract with _deploy method.
2. invoke the contract with method.
3. send neo/gas, invokefunction with transfer neo script.

Testing MethodToken is not awailable as neo-devpack-dotnet is not ready yet.

File: src/neo/SmartContract/ApplicationEngine.cs [hunk 1]
```diff
@@ -106,6 +106,7 @@ private ExecutionContext CallContractInternal(UInt160 contractHash, string metho
             CallFlags callingFlags = state.CallFlags;
 
             if (args.Length != md.Parameters.Length) throw new InvalidOperationException($"Method {method} Expects {md.Parameters.Length} Arguments But Receives {args.Length} Arguments");
+            if (hasReturnValue ^ (md.ReturnType != ContractParameterType.Void)) throw new InvalidOperationException();
```
Comment (cloud8little) [inline src/neo/SmartContract/ApplicationEngine.cs:109]: can we add exception message?

Comment (erikzhang) [reply src/neo/SmartContract/ApplicationEngine.cs]: Done.

**Comment by erikzhang** [comment]
Go?

**Pull request merged**